src/Widgets/DisplayWidget.vala

```diff
@@ -33,10 +34,22 @@ public class Network.Widgets.DisplayWidget : Gtk.Box {
         image.icon_name = "network-wired-symbolic";
         image.icon_size = Gtk.IconSize.LARGE_TOOLBAR;
 
+        extra_info_label = new Gtk.Label (null);
+        extra_info_label.margin_left = 4;
+        extra_info_label.margin_bottom = 4;
+        extra_info_label.valign = Gtk.Align.END;
+        extra_info_label.vexpand = true;
+        extra_info_label.no_show_all = true;
+        extra_info_label.visible = false;
```

Comment:
Can we use a revealer instead of setting the visible property? This way we get a nice animation when changing network types instead of things just popping into a new location